pkgs/applications/networking/mastodon-bot/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, pkgs }:
+(import ./composition.nix {
+  inherit pkgs;
+  inherit (stdenv.hostPlatform) system;
+})."package".overrideAttrs (old: {
+  buildInputs = old.buildInputs ++ [ pkgs.makeWrapper pkgs.lumo ];
+  postInstall = ''
+    # remove symlink to node_modules/.bin
+    rm $out/bin
+    mkdir $out/bin
+
+    # link to lumo executable
+    patchShebangs $out/lib/node_modules/.bin/mastodon-bot
```

Comment:
Please move this to `pkgs/development/node-packages`.
In `default-v10.nix` you can see how to override packages.

Reply:
Thanks, having a look!

Reply:
First did it manually to test it works - it does!

Converting this PR to a draft until mastodon-bot is published to npmjs.org so we can autogenerate most of the nixpkgs changes (https://github.com/yogthos/mastodon-bot/issues/39).

Thanks for the nudge!